data/libs/autoload.lua

```diff
@@ -8,6 +8,16 @@ string.trim = function(s)
 	return string.gsub(s or "", "^%s*(.-)%s*$", "%1")
 end
 
+string.capitalizeSentences = function (s)
```

Comment:
Note that Lua APIs are invariably one or two words with no capitalization or underscores. Thus this should be `string.capitalize` or `string.sentencecase`. I'd prefer the latter, as the former should be reserved for a function which capitalizes the leading letters of all words in the string.

(If you can find a clear name that is even shorter, it would be preferred!)

Reply:
string.scase works for me.